src/containers/home.jsx

```diff
@@ -22,6 +23,7 @@ class Home extends Component {
     const View = isLoggedIn ? LoggedInHome : LoggedOutHome;
     return (
       <div>
+        <TitleBar/>
```

Comment:
your title bar always has a `login` button, whether you're logged in or not.

i was suggesting letting `title_bar` take in `left` and `right` props because then `logged_in` and `logged_out` could determine what links to render.